shell/platform/windows/window_win32.cc

```diff
@@ -511,6 +515,23 @@ HWND WindowWin32::GetWindowHandle() {
   return window_handle_;
 }
 
+float WindowWin32::GetScrollOffsetMultiplier() { 
+  return scroll_offset_multiplier_; 
+}
+
+void WindowWin32::UpdateScrollOffsetMultiplier() {
+  UINT lines_per_scroll;
+  // Get lines per scroll wheel value from Windows
+  if (SystemParametersInfo(SPI_GETWHEELSCROLLLINES, 0, &lines_per_scroll, 0) == 0) {
+    // Windows default
+    lines_per_scroll = 3;
```

Comment:
Consider adding a `kLinesPerScrollWindowsDefault` constant

Reply:
Added in commit d50f423b42d9d3dde82ba3ee1419192d4d97c27c